dynamic-amm-quote/src/error.rs

```diff
@@ -0,0 +1,179 @@
+//! Error module includes error messages and codes of the program
+use anchor_lang::prelude::*;
+
+/// Error messages and codes of the program
+#[error_code]
+pub enum PoolError {
```

Comment:
I think this should be moved into `programs/dynamic-amm` in case the on chain integrated program wish to handle the error.

Reply:
fixed